### PR TITLE
Add L10N for GameCollections.cs

### PR DESCRIPTION
**Pull request by Flactine**

### CODE REVIEW

**Review by MahBoiDeveloper — APPROVED** [review]

File: ClientCore/CnCNet5/GameCollection.cs [hunk 1]
```diff
@@ -202,7 +203,7 @@ public void Initialize()
                 {
                     ChatChannel = "#cncnet",
                     InternalName = "cncnet",
-                    UIName = "General CnCNet Chat",
+                    UIName = "General CnCNet Chat".L10N("Client:Core:GeneralCnCNetChat"),
```
[the remark this answers is not in the document]
Comment (SadPencil) [reply ClientCore/CnCNet5/GameCollection.cs:206]: e.g., Mental Omega -> 心灵终结 (Chinese)

**Review by Metadorius — APPROVED** [review]

**Comment by enderseven1** [comment]
I have translated these names. But i set the L10N in a function names GetUIName(?) you can see my client code. Because some strange bug maybe happens if you set L10N at that UIName="".

**Comment by MahBoiDeveloper** [comment]
> I have translated these names. But i set the L10N in a function names GetUIName(?) you can see my client code. Because some strange bug maybe happens if you set L10N at that UIName="".

Do you have a screenshot of the bug you described?

**Comment by MahBoiDeveloper** [comment]
Translation works fine for me.

Translated names of games/mods:
image

Options works fine:
image

General CnCNet chat is displayed normal:
image

Yuri's Revenge chat too:
image

**Comment by enderseven1** [comment]
I forgot other details.
The bug maybe happens on option panel.
But the bug doesn't happen now, it's good.

**Pull request merged**